paper/src/main/java/com/denizenscript/denizen/paper/events/PlayerInventorySlotChangeScriptEvent.java

```diff
@@ -0,0 +1,92 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import io.papermc.paper.event.player.PlayerInventorySlotChangeEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class PlayerInventorySlotChangeScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player inventory slot changes
+    //
+    // @Group Paper
+    //
+    // @Location true
+    //
+    // @Plugin Paper
+    //
+    // @Switch from:<item> to only process the event if the previous item in the slot matches the specified item.
+    // @Switch to:<item> to only process the event if the new item in the slot matches the specified item.
+    // @Switch slot:<slot> to only process the event if a specific slot was clicked.
+    //
+    // @Triggers when the item in a slot of a player's inventory changes.
+    // Note that this fires for every item in the player's inventory when they join.
+    //
+    // @Context
+    // <context.new_item> returns an ItemTag of the new item in the slot.
+    // <context.old_item> returns an ItemTag of the previous item in the slot.
+    // <context.slot> returns an ElementTag(Number) of the slot that was changed.
+    // <context.raw_slot> returns an ElementTag(Number) of the raw number of the slot that was changed.
+    // -->

```

Comment:
Should have `@Player Always.` in the meta

Reply:
My bad, added